### PR TITLE
address UDP and non-TCP traffic suitability of MP-DCCP

### LINKED ISSUE #122: UDP and plain IP
The abstract says the following:
```
Compared to existing multipath protocols such as MPTCP, MP-
   DCCP provides specific support for non-TCP user traffic as UDP or
   plain IP
```

while there is nothing in the draft which explains how this is provided.

### PR DESCRIPTION
**Pull request by dirkhugo**
closes #122

### CODE REVIEW

**Comment by dirkhugo** [comment]
proposed sentence in intro enough?

**Comment by markusa** [comment]
@boucadair : Does this address your comment, Med?

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 1]
```diff
@@ -200,6 +200,10 @@ MP-DCCP can be applied for load-balancing, seamless session handover, and
 aggregation purposes (referred to as Access Traffic Steering, Switching, and Splitting (ATSSS)
 in the 3GPP terminology {{TS23.501}}).
 
+As pointed out in {{I-D.amend-tsvwg-multipath-framework-mpdccp}} the proposed encapsulation in terms of 
+lightweight DCCP flow headers is more appropriate for unreliable 
+IP traffic in terms of UDP and other non-TCP packets in comparison to MPTCP.
+
```
Comment (boucadair) [inline draft-ietf-tsvwg-multipath-dccp.mkd:206]:
```suggestion
Such considerations are not detailed in the present specification. 
```

**Comment by boucadair** [comment]
See the proposed change to the PR proposed by Dirk. I think we can basically declare this out of scope from this document and that further assessment to be conducted in companion documents.

**Pull request merged**